File: 1064.py
a = float(input())
b = float(input())
c = float(input())
d = float(input())
e = float(input())
f = float(input())
s=0
m =0
t =[a,b,c,d,e,f]
for n in t:
  if n > 0:
    s = s + 1
    m = m + (n)
print('{} valores positivos'.format(s))
print('{:.1f}'.format(m/s))

File: CSharp/1064.cs
using System;
using System.Globalization;

namespace teste
{
    class Program
    {
        static void Main(string[] args)
        {
            CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
            decimal media=0;
            int cont=0;
            decimal n;

            for(int i =0;i<6;i++)
            {
                n = decimal.Parse(Console.ReadLine());
                if(n>0)
                {
                    cont++;
                    media+=n;
                }
            }
            Console.WriteLine($"{cont} valores positivos");
            Console.WriteLine($"{(decimal)media/cont:F1}");
        } 
}
}
